src/org/labkey/test/components/ui/navigation/apps/ServerNotificationItem.java

```diff
@@ -0,0 +1,218 @@
+package org.labkey.test.components.ui.navigation.apps;
```

Comment:
This should match the component package.
```suggestion
package org.labkey.test.components.ui.notifications;
```

Reply:
Done.